src/parsers/manifest/smooth/representationIndex.ts

```diff
@@ -437,7 +438,7 @@ export default class SmoothRepresentationIndex
         return false;
       }
 
-      const repeat = lastSegmentInTimeline.r || 0;
+      const repeat = lastSegmentInTimeline.r || 1;
```

Comment:
Huh? repeat === 0 means 1 segment. repeat === 1 means two segments. If repeat is not defined it would means there's only 1 IMO